Exec/RegTests/IsothermalSoretChannel/isothermal-soret.inp

```diff
@@ -86,6 +86,6 @@ amr.gradT.adjacent_difference_greater = 200
 amr.gradT.field_name = temp
 
 #---------------------- Debug/HPC CONTROL-------------------------
-amrex.fpe_trap_invalid = 1
-amrex.fpe_trap_zero = 1
-amrex.fpe_trap_overflow = 1
+#amrex.fpe_trap_invalid = 1
```

Comment:
Are these options a thing anymore?

Reply:
Yes they still are. Sometimes they cause false postives on Macs though, so I had them commented out and accidentally left that when committing.

Reply:
Actually for testing the proper system-dependent settings for those flags are specified here, so I'm lkeeping them commented out in the input file: https://github.com/AMReX-Combustion/PeleLMeX/blob/176deb4dfcb4d0ba306f0206f32fdd9436d2cad1/Tests/CMakeLists.txt#L62